docs/developers/extensions.rst

```diff
@@ -0,0 +1,49 @@
+.. _developers_extensions:
+
+==========
+Extensions
+==========
+
+Keycloak token exchange extension
+=================================
+
+The municipality of Den Haag started a proof of concept for the addition of Attribute Based Access Control (ABAC) to the
+ZGW APIs. This means that an authenticated user should only be able to retrieve data from the ZGW APIs which they are
+entitled to see. In the proof of concept, the Keycloak preview feature called "`token exchange`_" is leveraged to get a
+OAUTH2 access token that is then sent to the downstream APIs to determine what the user is allowed to retrieve.
```

Comment:
```suggestion
(OAUTH2) access token that is then sent to the downstream APIs to determine what the user is allowed to retrieve.
```

it doesn't necessarily have to be an OAUTH access token, as long as it uses a Bearer token in the `Authorization` header, and even then there's some flexibility to use different headers or token formats (at least on our end).

If keycloak is configured to return a ZGW API's JWT, we can use that as is!